docusaurus/docs/what-is-included/internationalization.md

```diff
@@ -15,6 +15,8 @@ We have chosen [`react-intl`](https://github.com/formatjs/react-intl/) for inter
 
 The integration of `react-intl` into Next.js is made with [@moxy/next-intl](https://github.com/moxystudio/next-intl).
 
+ℹ️ If you are really sure internationalization is not needed in your project, you'll want to remove all the unnecessary `intl` related code. Start by uninstalling `react-intl` and `@moxy/next-intl`. You can then proceed to do a global search for both `react-intl` and `@moxy/next-intl` to find all their occurrences across the project, and clean the places where they are not being used anymore.
```

Comment:
I think we should mention that we should remove the root `intl` folder. Also, we no longer need the `full-icu` package and it’s usage in the npm scripts in the package.json.